src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -372,14 +381,13 @@ def remove_pods(self, names):
                     self.rook_env.namespace,
                     body=client.V1DeleteOptions()
                 )
-                num += 1
-        return "Removed %d pods" % num
+        return [f'Removed Pod n' for n in names]
```

Comment:
Let it just return pod names ```[f'n' for n in names]```

Reply:
do you mean to say 
```suggestion
        return [n for n in names]
```
?

Reply:
> do you mean to say
> 
> ?

Yes

Reply:
this is the corresponding string in cephadm *per daemon*

https://github.com/ceph/ceph/blob/ae167bd496d7dc0a72a24d7c393c32c5420c6bc2/src/pybind/mgr/cephadm/serve.py#L858